github/users.tf

```diff
@@ -92,6 +92,17 @@ module romain_maliachauguste {
   username = "R0one"
 }
 
+module thibaud_duhautbout {
+  source   = "./users"
+  username = "tduhautb"
+  teams = {
+    "flairsou" = {
+      team = github_team.flairsou.id,
+      role = "maintainer"
```

Comment:
Toujours pas fan des maintainers quand un member suffit mais why not

Reply:
J'aimerais bien que Thibaud puisse merge aussi, et il sait gérer Git ;)

Reply:
C'est pas le statut maintainer qui gere ca, mais ok why not ca marche aussi ^^